src/main/resources/templates/teams.html

```diff
@@ -75,6 +97,9 @@ <h2 class="team-heading team-heading-current-team" th:text="#{team.heading.reque
 	                                <!--<td><a href="#" title="Accept" class="btn btn-success" th:href="@{/accept_participation/{teamId}(teamId=${teamEntry.id})}"><i class="fa fa-check-circle-o fa-lg"></i> Accept</a><a href="#" title="Ignore" class="btn btn-warning" th:href="@{/ignore_participation/{teamId}(teamId=${teamEntry.id})}"><i class="fa fa-times-circle-o fa-lg"></i> Ignore</a></td>-->
 	                            <!--</tr>-->
 	                        <!--</table>-->
+                            <!--<tr>-->
+                                <!--<td height="700"></td>-->
+                            <!--</tr>-->
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AAvoidCommentedOutCodeCheck)